RecoHI/HiMuonAlgos/python/HiRegionalRecoMuon_cff.py

```diff
@@ -47,21 +47,32 @@
 
 # replace the new names
 
-remuonIdProducerSequence = cms.Sequence(reglbTrackQual*remuons*remuonEcalDetIds*remuonShowerInformation)
-remuIsoDeposits_muons    = cms.Sequence(remuIsoDepositTk+remuIsoDepositCalByAssociatorTowers+remuIsoDepositJets)
-remuIsolation_muons      = cms.Sequence(remuIsoDeposits_muons)
-remuIsolation            = cms.Sequence(remuIsolation_muons)
-#run this if there are no STA muons in events
-muontracking                        = cms.Sequence(standAloneMuonSeeds * standAloneMuons * hiRegitMuTracking * reglobalMuons)
+remuonIdProducerTask     = cms.Task(reglbTrackQual,remuons,remuonEcalDetIds,remuonShowerInformation)
+remuonIdProducerSequence = cms.Sequence(remuonIdProducerTask)
 
-#the default setting assumes the STA is already in the event
-muontracking_re                     = cms.Sequence(hiRegitMuTracking * reglobalMuons)
-muontracking_with_TeVRefinement_re  = cms.Sequence(muontracking_re * retevMuons)
+remuIsoDeposits_muonsTask= cms.Task(remuIsoDepositTk,remuIsoDepositCalByAssociatorTowers,remuIsoDepositJets)
+remuIsoDeposits_muons    = cms.Sequence(remuIsoDeposits_muonsTask)
```

Comment:
This was only needed here internally: sequence not needed any more now, please remove